src/go/rpk/pkg/cli/cluster/storage/list-mountable.go

```diff
@@ -0,0 +1,85 @@
+package storage
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/redpanda-data/common-go/rpadmin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/adminapi"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newListMountable(fs afero.Fs, p *config.Params) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list-mountable",
+		Short: "List mountable topics from object storage",
+		Long: `List topics that are available to mount from object storage.
+
+This command displays topics that exist in object storage and can be mounted
+to your Redpanda cluster. Each topic includes its location in object storage
+and namespace information if applicable.`,
+		Example: `
+List all mountable topics:
+	rpk cluster storage list-mountable
```

Comment:
```suggestion
  rpk cluster storage list-mountable
```
Just to avoid tabs like in the other commands

Reply:
done